uint8 FrameHead
float64 BulletSpeed
uint8 MyColor
uint8 VisionMode
float64 IMURaw
float64 IMUPitch
float64 IMUYaw
uint8 FrameTail